src/App.tsx

```diff
@@ -11,6 +12,7 @@ const App = () => {
         <Route path="/events" element={""} />
```

Comment:
I think we will need to change the `/home` endpoint for now to `/` as on the [pre-deploy-site](https://union-client-project--pr9-11-feature-404-not-f-lzjhsx3p.web.app/) the 404 page is being shown